src/main/resources/com/cloudbees/jenkins/plugins/sshagent/SSHAgentStep/help.html

```diff
@@ -0,0 +1,7 @@
+<pre><code>
+node {
+  sshagent ([ignoreMissing: false, credentials: ['14714940-3b66-48f1-bf60-7ec8ec7c5581']]) {
```

Comment:
@jglick I'll review the help information but let me not to include the part of *Snippet Generator* in this PR. I'll try to do later.